pywbem_mock/_mainprovider.py

```diff
@@ -1009,24 +1092,30 @@ def CreateClass(self, **params):
     #
     ####################################################################
 
-    def ModifyClass(self, **params):
-        # pylint: disable=invalid-name,unused-argument
+    def ModifyClass(self, namespace, ModifiedClass):
+        # pylint: disable=unused-argument
```

Comment:
This pylint marker is not needed once the method is implemented. It does not hurt that pylint reports an unused argument as long as it is not yet implemented.

Reply:
Removed several of them.  DONE